examples/docker-deployment/README.md

```diff
@@ -0,0 +1,122 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Wrapping a Model Deployment with Docker
+
+This README outlines the many options you have for storing a deepsparse model inside
+a docker container.
+
+The [Dockerfile](Dockerfile) shows one case of this: running `sparsezoo.download`
+when building the image, and passing to `deepsparse.server task`. Example usage:
+
+```bash
+docker build --build-arg TASK=qa --build-arg MODEL_STUB=zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned95_obs_quant-none -t qa .
+```
+
+## What is a Model Deployment?
+
+It is a directory with at least a `model.onnx` file in it, but usually other configuration files.
```

Comment:
In the future it would be great to link to a sample deployment folder in sparsezoo.